tests/testthat/test-module_teal.R

```diff
@@ -602,11 +602,13 @@ testthat::describe("srv_teal teal_modules", {
         ),
         expr = {
           session$setInputs("teal_modules-active_tab" = "module_1")
+          session$flushReact()
```

Comment:
@averissimo can you remind me what `session$flushReact()` is doing?

Reply:
It is used only in testing and it is used to trigger the reactive chain.

In cases like this the reactivity is flushed due to the input change and it might be to correct an odd behavior or to be safe.

An example where it is absolutely needed is when testing the `srv_teal_transform_data`. The `observeEvent` inside the `.trigger_on_success(data_handled)` call is only triggered after the flush.

Reply:
ok thanks for the explanation!